src/main/resources/view/PersonListCard.fxml

```diff
@@ -29,24 +29,24 @@
           </minWidth>
         </Label>
         <Hyperlink fx:id="name" text="\$first" styleClass="cell_big_label" onAction="#openTelegram"/>
+        <Label fx:id="group" styleClass="cell_small_label" text="\$group" prefWidth="250" />
+        <Region HBox.hgrow="ALWAYS" />
       </HBox>
       <VBox spacing="3">
         <FlowPane hgap="5" alignment="CENTER_LEFT">
           <Label styleClass="cell_small_label_tags" text="🏷 Tags:" prefWidth="175" />
           <FlowPane fx:id="tags" />
         </FlowPane>
         <FlowPane hgap="5" alignment="CENTER_LEFT">
-          <Label styleClass="cell_small_label_tags" text="🗒 Comments:" prefWidth="175" />
+          <Label styleClass="cell_small_label_tags" text="🗒️ Comments:" prefWidth="200" />
```

Comment:
We should not re-introduce the regression of emotes causing display issues on Windows. While the `prefWidth` can change, we should not change the text representation.